src/Command/CommandHelper.php

```diff
@@ -129,7 +131,7 @@ public static function begin(
 			try {
 				$projectConfig = $loader->load($projectConfigFile, null);
 			} catch (\Nette\InvalidStateException | \Nette\FileNotFoundException $e) {
-				$errorOutput->writeln($e->getMessage());
+				$application->renderException($e);
```

Comment:
Why this change? The message is usually about which file wasn't found which is sufficient.